### PR TITLE
Resolve locations in Faros Destination for Google Calendar events

### LINKED ISSUE #216: Resolve locations in Faros Destination for Google Calendar events
Resolve locations using Faros `/geocoding/lookup` API - https://dev.api.faros.ai/docs.html#Geocoding

### PR DESCRIPTION
**Pull request by Roma-Kyrnis**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix #216 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Phabricator Source | [sources/phabricator-source](sources/phabricator-source) |
 SquadCast Source | [sources/squadcast-source](sources/squadcast-source) | `docker pull farosai/airbyte-squadcast-source` | [![](https://img.shields.io/github/v/tag/faros-ai/airbyte-connectors?label=)](https://hub.docker.com/orgs/farosai/repositories)
 Statuspage Source | [sources/statuspage-source](sources/statuspage-source) | `docker pull farosai/airbyte-statuspage-source` | [![](https://img.shields.io/github/v/tag/faros-ai/airbyte-connectors?label=)](https://hub.docker.com/orgs/farosai/repositories)
 VictorOps Source | [sources/victorops-source](sources/victorops-source) | `docker pull farosai/airbyte-victorops-source` | [![](https://img.shields.io/github/v/tag/faros-ai/airbyte-connectors?label=)](https://hub.docker.com/orgs/farosai/repositories)
+GoogleCalendar Source | [sources/googlecalendar-source](sources/googlecalendar-source) | `docker pull farosai/airbyte-googlecalendar-source` | [![](https://img.shields.io/github/v/tag/faros-ai/airbyte-connectors?label=)](https://hub.docker.com/orgs/farosai/repositories)
```
Comment (tovbinm) [inline README.md:22]: it's already on the list. remove this.

File: destinations/faros-destination/src/converters/googlecalendar/calendar_models.ts [hunk 2]
```diff
@@ -1,4 +1,62 @@
-export interface Schema$Event {
+interface Location {
```
Comment (tovbinm) [inline destinations/faros-destination/src/converters/googlecalendar/calendar_models.ts:1]: you dont need to redefine the types. they are already exported from the feeds-sdk

Comment (tovbinm) [reply destinations/faros-destination/src/converters/googlecalendar/calendar_models.ts:1]: just import them from there

File: destinations/faros-destination/src/converters/googlecalendar/calendar_models.ts [hunk 3]
```diff
@@ -1,4 +1,62 @@
...
+    country: string;
+    countryCode: string;
+  };
+  coordinates: {
+    lat: number;
+    lon: number;
+  };
+}
+
+const location_geocode: Location = {
```
Comment (tovbinm) [inline destinations/faros-destination/src/converters/googlecalendar/calendar_models.ts:24]: move the test value to test folder

File: sources/googlecalendar-source/acceptance-test-config.yml [hunk 4]
```diff
@@ -21,9 +21,9 @@ tests:
   full_refresh:
     - config_path: "secrets/config.json"
       configured_catalog_path: "test_files/full_configured_catalog.json"
-      ignored_fields:
-        events:
-          - nextSyncToken
+      # ignored_fields:
```
Comment (tovbinm) [inline sources/googlecalendar-source/acceptance-test-config.yml:24]: ??

**Review by tovbinm — CHANGES_REQUESTED** [review]
Geocode API invocation should be done in Faros Destination, and not in Google Calendar source. https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/faros-destination/src/converters/googlecalendar/events.ts#L77

File: README.md (hunk 1 above)
Comment (Roma-Kyrnis) [reply README.md:22]: removed

File: sources/googlecalendar-source/acceptance-test-config.yml (hunk 4 above)
Comment (Roma-Kyrnis) [reply sources/googlecalendar-source/acceptance-test-config.yml:24]: I'll remove this line

**Comment by Roma-Kyrnis** [comment]
Ok. I'll remove Geocode from source.

Where do i use the async function to pull Geocode in destination's converter?

**Comment by tovbinm** [comment]
suppressed by https://github.com/faros-ai/airbyte-connectors/pull/230

**Comment by tovbinm** [comment]
@Roma-Kyrnis please pick up the work on https://github.com/faros-ai/airbyte-connectors/pull/230 and finish it up.

**Pull request closed without merging**